cobalt/watchdog/watchdog.h

```diff
@@ -107,6 +108,11 @@ class Watchdog : public Singleton<Watchdog> {
   bool GetPersistentSettingWatchdogCrash();
   void SetPersistentSettingWatchdogCrash(bool can_trigger_crash);
 
+  // LogTrace API
```

Comment:
Link to docs, idl reference, and anything else that could be useful to determine what the LogTrace API is.